front/src/views/user/Housing.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
+    <stepper></stepper>
+  </div>
+</template>
+
+<script>
+// @ is an alias to /src
+import { Stepper } from '@components';
+
+export default {
+  components: {
+    Stepper
```

Comment:
lets use prefix mtr for our custom components
so, MtrStepper

Reply:
renamed MmtStepper